hack/clean-e2e.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+cd "${REPO_ROOT}" || exit 1
+
+minikube delete
+docker stop vbmc
+docker rm vbmc
+virsh destroy --domain bmo-e2e-0
+virsh undefine --domain bmo-e2e-0 --remove-all-storage
+virsh net-undefine baremetal-e2e
+
+rm -rf "${REPO_ROOT}/test/e2e/_artifacts"
```

Comment:
You are already in repo root by L4.

Reply:
Well yes, but no harm being explicit :sweat_smile: 
Also saves us if someone happens to put a `cd` somewhere between L4 and L13 and not notice this :wink: